test/e2e/specs/splitting-merging.test.js

```diff
@@ -63,7 +63,7 @@ describe( 'splitting and merging blocks', () => {
 		await page.keyboard.press( 'Backspace' );
 
 		// Replace contents of first paragraph with "Bar".
-		await pressTimes( 'Backspace', 3 );
+		await pressTimes( 'Backspace', 4 );
```

Comment:
That's surprising, any ideas why it needs to be updated?

Reply:
I don't know that its strictly an issue here, but caret placement in and around inline boundaries has been historically problematic and inconsistent and we should be careful to avoid worsening this. e.g. in some cases depending on internal state of TinyMCE, caret will still move inside inline boundary when backspacing (e.g. if Enter + Backspace when at end of a paragraph inside inline boundary).

Related: #9040 (internal state not an issue if its not preserved)